pkg/deploy/assets/rp-production.json

```diff
@@ -378,9 +378,9 @@
                     },
                     "storageProfile": {
                         "imageReference": {
-                            "publisher": "RedHat",
-                            "offer": "RHEL",
-                            "sku": "8-LVM",
+                            "publisher": "MicrosoftCBLMariner",
+                            "offer": "cbl-mariner",
+                            "sku": "cbl-mariner-2-fips",
```

Comment:
Let's use `MicrosoftCBLMariner:cbl-mariner:cbl-mariner-2-gen2-fips:latest`

https://eng.ms/docs/products/azure-linux/gettingstarted/azurevm/azurevm

Reply:
I agree. I had plans to update this but haven't gotten there yet.
I'll update now and start testing with the gen2 image.